include/sgx/faasm_sgx_error.h

```diff
@@ -1,12 +1,7 @@
-//
-// Created by Joshua Heinemann on 10.06.20.
-// TU-Braunschweig (heineman@ibr.cs.tu-bs.de)
-//
-
-#ifndef FAASM_SGX_FAASM_ERROR_H
-#define FAASM_SGX_FAASM_ERROR_H
+#pragma once
```

Comment:
The convention in this repo is to use `#pragma once`.

We also don't have a standard file header or author names. Code committed to the repo will fall under the licence (Apache 2.0)